src/game/lara.h

```diff
@@ -119,6 +120,7 @@ void LaraColPushBlock(ITEM_INFO *item, COLL_INFO *coll);
 void LaraColPullBlock(ITEM_INFO *item, COLL_INFO *coll);
 void LaraColPPReady(ITEM_INFO *item, COLL_INFO *coll);
 void LaraColPickup(ITEM_INFO *item, COLL_INFO *coll);
+void LaraColControlled(ITEM_INFO *item, COLL_INFO *coll);
```

Comment:
I'd rename `Controlled` to something else, `WalkToItem` or something.

Reply:
Is it ok to leave it as is? This name matches the TR5 codebase's name.

Reply:
OK let's keep it then.